lib/geared_pagination/page.rb

```diff
@@ -35,7 +35,11 @@ def only?
     end
 
     def last?
-      number >= recordset.page_count
+      number == recordset.page_count
+    end
+
+    def has_more_pages_after_this?
```

Comment:
Would try for a short adjective matching the other methods' style. Maybe just `#more?` ?

Reply:
Went with `#before_last?` by @georgeclaghorn's suggestion, which I think is quite good, maybe clearer than `#more?`, even if `#more?` matches the other methods' style better.